deps/rabbit/test/feature_flags_SUITE.erl

```diff
@@ -112,59 +112,53 @@ end_per_suite(Config) ->
 init_per_group(feature_flags_v1, Config) ->
     rabbit_ct_helpers:set_config(Config, {enable_feature_flags_v2, false});
 init_per_group(feature_flags_v2, Config) ->
-    rabbit_ct_helpers:set_config(Config, {enable_feature_flags_v2, true});
+    %% Before we run `feature_flags_v2'-related tests, we must ensure that both
+    %% umbrellas support them. Otherwise there is no point in running them. The
+    %% `feature_flags_v1' group already covers testing in that case.
+    case rabbit_ct_helpers:get_config(Config, secondary_umbrella) of
+        undefined ->
+            rabbit_ct_helpers:set_config(Config, {enable_feature_flags_v2, true});
+        _ ->
+            %% To determine that `feature_flags_v2' are supported, we can't
+            %% query RabbitMQ which is not started. Therefore, we check if the
+            %% source of `rabbit_ff_controller' is present.
+            Dir1 = ?config(rabbit_srcdir, Config),
+            Dir2 = ?config(secondary_rabbit_srcdir, Config),
```

Comment:
I'm going to validate that this method of checking will also work with bazel, as there are not secondary sources present